src/me/hkgumbs/tictactoe/test/java/simulation/DefaultControllerTest.java

```diff
@@ -111,49 +101,16 @@ public void haltPlayerInputThenCheckInProgress() {
     }
 
     @Test
-    public void haltPlayerThenCheckCompleted() {
-        saveDefaultSimulation("n\n5\n3\n");
-        try {
-            controller.startSimulation();
-        } catch (NoSuchElementException e) {
-        }
-
-        assertEquals(Simulation.State.COMPLETED, simulation.getState());
-        assertInputStreamEmpty();
-    }
-
-    @Test
-    public void haltPlayerThenCheckInitialAgain() {
-        saveDefaultSimulation("n\n5\n3\ny\n");
+    public void haltPlayerThenCheckReplayInProgress() {
         try {
+            saveSimulation("0\n6\n1\n7\n2\n");
             controller.startSimulation();
+        } catch (Configuration.CannotApplyException e) {
+            fail(HUMAN_ERROR);
         } catch (NoSuchElementException e) {
         }
 
-        assertEquals(Simulation.State.INITIAL, simulation.getState());
+        assertEquals(Simulation.State.IN_PROGRESS, simulation.getState());
         assertInputStreamEmpty();
     }
-
-    @Test
-    public void fullSimulation() {
-        saveDefaultSimulation("n\n5\n3\nn\n");
-        assertSimulationState(Simulation.State.TERMINATED);
-    }
-
-    @Test
-    public void fullSimulatioWithReplay() {
-        saveDefaultSimulation("n\n5\n3\ny\nn\n5\n3\nn\n");
-        assertSimulationState(Simulation.State.TERMINATED);
-    }
-
-    @Test
-    public void fullSimulationWithTwoMinimax() {
-        String[] args = new String[]{"--minimax"};
-        try {
-            saveDefaultSimulation("", args);
-        } catch (Configuration.CannotApplyException e) {
-            fail("--minimax should be a valid option");
-        }
-        assertSimulationState(Simulation.State.TERMINATED);
-    }
```

Comment:
Because I rebased, Colin's comments got deleted, but I was notified about this:
>>> A few tests got deleted here - what's the reason? Do you lose any confidence in the test suite as a result? Why or why not?

The tests I deleted became redundant when I moved the turn-ordering functionality to PlayerConfiguration. Also testing with human v human (instead of human v minimax) makes the tests more obvious and eliminates redundancy between this class and PlayerConfigurationTest